### PR TITLE
Added --plyr-video-background for having control over the background …

### PR DESCRIPTION
**Pull request by nepomuc**
…of a video with alpha channel (webm) or a poster image with alpha channel.

### Checklist
- [✅] Use `develop` as the base branch
- [✅] Exclude the gulp build (`/dist` changes) from the PR
- [💔] Test on [supported browsers](https://github.com/sampotts/plyr#browser-support) ==> Couldn't try it on Apple devices, since I don't own any.

### CODE REVIEW

File: src/sass/settings/colors.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // ==========================================================================
 
 $plyr-color-main: var(--plyr-color-main, hsl(198, 100%, 50%)) !default;
+$plyr-video-background: var(--plyr-video-background, rgba(0,0,0,1)) !default;
```
Comment (sampotts) [inline src/sass/settings/colors.scss:6]: Missing spaces between the RGB values - did you run prettier? It should be `rgba(0, 0, 0, 1)`

**Review by sampotts — COMMENTED** [review]
Thanks for this. Will just use this PR and close the other. PRs go through develop currently.

File: src/sass/settings/colors.scss (hunk 1 above)
Comment (nepomuc) [reply src/sass/settings/colors.scss:6]: I ran the gulp "build" job and assumed that it was running everything that's neccessary. I didn't run prettier in addition to that.

Comment (sampotts) [reply src/sass/settings/colors.scss:6]: No dramas. I can sort that out. It wasn't obvious in the docs anyhow.

**Review by sampotts — APPROVED** [review]

**Pull request merged**